R/model_WRMF.R

```diff
@@ -358,6 +358,10 @@ WRMF = R6::R6Class(
         res = float(0, nrow = private$rank, ncol = nrow(x))
       }
 
+      if (private$with_user_item_bias) {
+        res[1, ] = ifelse(private$precision == "double", 1.0, float::fl(1.0))
```

Comment:
You might want to change this line: https://github.com/wrathematics/float/issues/36